jabbercat/models.py

```diff
@@ -1,28 +1,27 @@
 import bisect
-import collections.abc
 import enum
 import typing
 import unicodedata
-
-import lxml.builder
-import lxml.etree
+from itertools import chain
+from typing import List
```

Comment:
I don’t like ``from typing import ...`` because the names are very generic.

Reply:
So I should rather use `typing.List[thing]` as type annotation?